docs/quickstart.md

```diff
@@ -63,33 +67,42 @@ Once you've completed the [Okta Provider Setup](okta_provider_setup.md) (alterna
 can skip to and complete Section 3 of the [Okta Provider Setup](okta_provider_setup.md) instead),
 follow through the below steps:
 
-**⚡️ Note:** Use `http://sso-auth.localtest.me` as the
+**⚡️ Note:** Use `http://sso-auth.localtest.me/okta/callback` as the
 **Authorized redirect URI** in step 3.
 
-At the end of step 3 you will be given a client ID and client secret which can be found at the bottom
-of the settings page for the Application you've created.
+At the end of step 3 you will be given a client ID and client secret, which can
+be found at the bottom of the settings page for the Application you've created.
 Create a new file called `env` with those values, like so:
 
-NOTE: `OKTAQUICKSTART` is a logical identifier that is used to group the configuration
-variables together for any one provider. This can be changed to an identifier that makes
-sense for your individual use case.
-
     PROVIDER_OKTAQUICKSTART_CLIENT_ID=<client id value>
     PROVIDER_OKTAQUICKSTART_CLIENT_SECRET=<client secret value>
-
-As well as these, you also need to make sure **sso** knows to use the 'Okta' provider
-(Okta is not the default provider), provide your Okta organisation URL (for example, 'sso-test.okta.com'),
-and give a provider 'slug' for the configuration of http routes.
-
-Add the below lines into the `env` file you just created:
-
-    DEFAULT_PROVIDER_SLUG=okta
     PROVIDER_OKTAQUICKSTART_OKTA_URL=<organisation url>
     PROVIDER_OKTAQUICKSTART_TYPE=okta
     PROVIDER_OKTAQUICKSTART_SLUG=okta
+    DEFAULT_PROVIDER_SLUG=okta
+
+Notes:
+
+- Start by copying `env.okta.example` to `env` (`cp env.okta.example env`)
```

Comment:
Having this `env.okta.example` file to copy seems like a good idea! Should we suggest to copy this file instead of `Create a new file called 'env' with those values, like so:` on line 75?

Reply:
Yeah, this was a bit awkward.  I've updated, lemme know what you think now!

Reply:
💯